docs/resources/cce_node.md

```diff
@@ -366,21 +357,9 @@ The following arguments are supported:
 * `runtime` - (Optional, String, ForceNew) Specifies the runtime of the CCE node. Valid values are *docker* and
   *containerd*. Changing this creates a new resource.
 
-* `extend_param` - (Optional, Map, ForceNew) Specifies the extended parameter.
+* `extend_params` - (Optional, List, ForceNew) Specifies the extended parameters.
+  The [object](#extend_params) structure is documented below.
   Changing this parameter will create a new resource.
-  The available keys are as follows:
-  + **agency_name**: The agency name to provide temporary credentials for CCE node to access other cloud services.
-  + **alpha.cce/NodeImageID**: The custom image ID used to create the BMS nodes.
-  + **dockerBaseSize**: The available disk space of a single docker container on the node in device mapper mode.
-  + **DockerLVMConfigOverride**: Specifies the data disk configurations of Docker.
```

Comment:
Is `DockerLVMConfigOverride` not supported in **extend_params**?

Reply:
it‘s deprecated in the API doc, `storage` is a better choice